db/migrate/20191126180559_create_test_results.rb

```diff
@@ -0,0 +1,15 @@
+class CreateTestResults < ActiveRecord::Migration[5.2]
+  def change
+    create_table :test_results, id: :uuid do |t|
+      t.datetime :start_time
+      t.datetime :end_time
+      t.decimal :score
+      t.references :profile, type: :uuid, index: true
+      t.references :host, type: :uuid, index: true
```

Comment:
I can add benchmark here, but what is the point? `test_result.profile.benchmark` will always get you the benchmark of the profile associated, which has to be the same one

Reply:
I guess the reason the XCCDF model has both is because it references both by ref_id, not by uuid. Since we're using uuids, we can keep the association relationship via profile like you have, but I'd also opt for adding that a TestResult `belongs_to :benchmark, through: profile`.

Reply:
That makes sense - will put it there